x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/telemetry/alerting_telemetry.ts

```diff
@@ -24,26 +24,18 @@ export default function createAlertingTelemetryTests({ getService }: FtrProvider
   const retry = getService('retry');
   const supertestWithoutAuth = getService('supertestWithoutAuth');
   const esTestIndexTool = new ESTestIndexTool(es, retry);
+  const esArchiver = getService('esArchiver');
 
   describe('alerting telemetry', () => {
     const alwaysFiringRuleId: { [key: string]: string } = {};
-    const objectRemover = new ObjectRemover(supertest);
 
     before(async () => {
-      // reset the state in the telemetry task
-      await es.update({
-        id: `task:Alerting-alerting_telemetry`,
-        index: '.kibana_task_manager',
-        body: {
-          doc: {
-            task: {
-              state: '{}',
-            },
-          },
-        },
-      });
+      await esArchiver.load('x-pack/test/functional/es_archives/event_log_telemetry');
```

Comment:
This is creating older event log index versions and populating some data into it so the telemetry runs across multiple event log indices, causing the reduce cycle to run in a meaningful way. If you run this functional test against `main`, you should see the telemetry fail with `search_phase_exception` whereas it passes on this branch.